ciphers/porta_cipher.py

```diff
@@ -0,0 +1,113 @@
+alphabet = {
```

Comment:
Please reformat this code with https://github.com/psf/black. Lines longer than 88 characters are going to get wrapped. 

Does this still work if these remain strings?
```python
     "A": ("ABCDEFGHIJKLM", "NOPQRSTUVWXYZ"),
```

Reply:
This works and formatting looks much better, added in latest changes